Tasks/KubernetesV1/src/kubernetescommand.ts

```diff
@@ -42,19 +42,35 @@ function getCommandOutputFormat(kubecommand: string) : string[] {
     return args;
 }
 
-function getCommandConfigurationFile() : string[] {
-    var args: string[] =[];
-    var useConfigurationFile : boolean  =  tl.getBoolInput("useConfigurationFile", false);
-    if(useConfigurationFile) {
-        var configurationPath = tl.getInput("configuration", true);
-        if(configurationPath && tl.exist(configurationPath))
-        {
+function getCommandConfigurationFile(): string[] {
+    var args: string[] = [];
+    var useConfigurationFile: boolean = tl.getBoolInput("useConfigurationFile", false);
+    let configurationPath = tl.getPathInput("configuration", false);
+    var inlineConfiguration = tl.getInput("inline", false);
+
+    if (!tl.filePathSupplied("configuration")) {
+        configurationPath = null;
+    }
+    if (configurationPath != null && inlineConfiguration != null) {
```

Comment:
should this also have a check that useConfigurationFile is true?

Reply:
This is done based on the feedback that during yaml configuration, `useConfigurationFile` flag may be omitted but path and/or inline configuration may be defined